src/extension/features/budget/display-target-goal-amount/index.js

```diff
@@ -29,12 +29,57 @@ export class DisplayTargetGoalAmount extends Feature {
 
   invoke() {
     this.addToolkitEmberHook('budget-table-row', 'didRender', this.addTargetGoalAmount);
+    this.addToolkitEmberHook('budget-table-row', 'didRender', this.calculateTotalGoals);
   }
 
   destroy() {
     $('.tk-target-goal-amount').remove();
   }
 
+  calculateTotalGoals() {
+    var categoryGoalTotalsDict = {};
+
+    $('.budget-table-row').each((_, element) => {
```

Comment:
I actually think we should just update `addTargetGoalAmount` to handle this.

basically, when you see `master-category-row` save off a total count and the current goal column element, then every time you see a sub category, update the total, if it's >0, write that new total to the goal column updating it each time the amount changes.

right now, this function would run a loop against all `budget-table-rows` for every single budget row which could be pretty bad on larger budgets